dashboard/src/components/AppView/CustomAppView/CustomAppView.test.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  InstalledPackageDetail,
+  AvailablePackageDetail,
+} from "gen/kubeappsapis/core/packages/v1alpha1/packages";
+import { getStore, mountWrapper } from "shared/specs/mountWrapper";
+import CustomAppView from ".";
+import { CustomComponent } from "../../../RemoteComponent";
+import { IAppViewResourceRefs } from "../AppView";
+
+const defaultState = {
+  config: { remoteComponentsUrl: "" },
+};
+
+const defaultProps = {
+  app: {
+    availablePackageRef: {
+      identifier: "bar",
+    },
+  } as InstalledPackageDetail,
+  resourceRefs: {
+    ingresses: [],
+    deployments: [
+      {
+        cluster: "default",
+        apiVersion: "apps/v1",
+        kind: "Deployment",
+        plural: "deployments",
+        namespaced: true,
+        name: "ssh-server-example",
+        namespace: "demo-namespace",
+      },
+    ],
+    statefulsets: [],
+    daemonsets: [],
+    otherResources: [
+      {
+        cluster: "default",
+        apiVersion: "v1",
+        kind: "PersistentVolumeClaim",
+        plural: "persistentvolumeclaims",
+        namespaced: true,
+        name: "ssh-server-example-root-pv-claim",
+        namespace: "demo-namespace",
+      },
+    ],
+    services: [
+      {
+        cluster: "default",
+        apiVersion: "v1",
+        kind: "Service",
+        plural: "services",
+        namespaced: true,
+        name: "ssh-server-example",
+        namespace: "demo-namespace",
+      },
+    ],
+    secrets: [],
+  } as unknown as IAppViewResourceRefs,
```

Comment:
Ill look into this. I cannot remember if there is a reason for this but Ill look. It might have been a weird case where it wasn't similar enough to the defined object.

Reply:
In the past, I've also used `Partial<MyType>`  (e.g., `as Partial<IAppViewResourceRefs>`) when I just wanted to define a subset of the properties of a type for a specific instance. 
More info: https://www.typescriptlang.org/docs/handbook/utility-types.html

Reply:
The TS error is kind of annoying since I straight up copied the object and I still got the error. I will try this solution instead of the `unknown`

Reply:
So I tried to leverage `Partial<>` and the compiler still did not like it so I left is as is :(